src/plugins/explore/public/components/query_panel/results_summary/results_summary.scss

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+@import "../shared";
+
+.exploreResultsSummary {
+  @include exploreQueryPanelButton;
+  @include exploreQueryPanelVerticalSeparator;
+}
+
+.exploreResultsSummary_popover_tooltip {
+  display: flex;
+  padding-inline-start: 2px;
+  transform: translate(-1px);
```

Comment:
nitpick: i had to google what this translate(singleArg) does. looks like it translates horizontally. Maybe just `translateX(-1px)` to be explicit?